### ISSUE TITLE
LTI v3.0.1 - Statistics page and attempt report displays incorrect time taken

### ISSUE THREAD

**Issue by peterlcsteele** [numbas/numbas-lti-provider#236]
The issue is demonstrated in the following screenshots. This installation is using Sydney/Australia (GMT+10) timezone - perhaps this is causing the issue.

Statistics page:
![statistics-time-taken-10hr](https://user-images.githubusercontent.com/12838081/169722623-d634ffef-0d71-48e8-9273-cd9e48f7d403.png)

Attempts:
![statistics-time-taken-attempts-download](https://user-images.githubusercontent.com/12838081/169722659-0dd8e44b-3162-4cfc-aa1d-8ca9bbd2d2a1.png)

**Comment by christianp** [comment]
"Tomorrow" is a mere 6 months later, but I've just had a look at this. I couldn't reproduce it, but I have found another exciting bug where the reported time spent was off by an hour. (see numbas/Numbas#17)

I've tried the following combinations of time zones:
* my PC UTC+0, LTI server UTC+11
* my PC UTC+11, LTI server UTC+0
* both PC and LTI server UTC+11.

The time was recorded correctly in every case. The only difference I can think of is that I'm using a local sqlite3 database, instead of postgresql. I can't easily set up a postgresql server in a different time zone.

Does this happen for every attempt, or just some? It looks like all of them, which suggests that there's just a timezone conversion failing to happen somewhere. I can't see where, though: the "time taken" is calculated on the client, and purely by subtracting `Date` objects created locally. 

What have you put for `TIME_ZONE` in `numbasltiprovider/settings.py`? What happens if you set your PC's time to UTC+0 and start an attempt?
